content/project/board-election-process.adoc

```diff
@@ -68,4 +52,44 @@ Like many things in the Jenkins community, the disclosure of corporate affiliati
 There are several motivations behind the above proposal:
 
 . Odd number of people prevents the tie problem
-. Given the low bar to the committership, we couldn't really come up with any criteria to decide who has the voting right. At the same time, we did want to restrict who can vote (as opposed to anyone unrelated to the project) for the stability.
\ No newline at end of file
+. Given the low bar to the committership, we couldn't really come up with any criteria to decide who has the voting right. At the same time, we did want to restrict who can vote (as opposed to anyone unrelated to the project) for the stability.
```

Comment:
Optional alternate phrasing?  Not blocking, and perfectly fine if this alternate is rejected.
```suggestion
. Given the low bar for permission to commit, we couldn't identify precise criteria to define the right to vote in board elections.  At the same time, we wanted to preserve stability by limiting voting rights to only those with some involvement in the project.
```